File: C_Sharp/1001-2000/p1732.cs
namespace p1732 {
    public class Solution {
        public int LargestAltitude(int[] gain) {
            int result = 0;
            int curr = 0;
            for(int i = 0; i < gain.Length; i++){
                curr += gain[i];
                result = int.Max(curr, result);
            }
            return result;
        }
    }    
}

File: C_Plus_Plus/1001-2000/p1732.cpp
#include <vector>
#include <algorithm>

using namespace std;

class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int result = 0;
        int curr = 0;
        for(int i = 0; i < gain.size(); i++){
            curr += gain[i];
            result = max(curr, result);
        }
        return result;
    }
};